pkg/assets/bindata.go

```diff
@@ -131,7 +131,7 @@ func assetsComponentsHostpathProvisionerClusterroleYaml() (*asset, error) {
 		return nil, err
 	}
 
-	info := bindataFileInfo{name: "assets/components/hostpath-provisioner/clusterrole.yaml", size: 609, mode: os.FileMode(436), modTime: time.Unix(1655736360, 0)}
+	info := bindataFileInfo{name: "assets/components/hostpath-provisioner/clusterrole.yaml", size: 0, mode: os.FileMode(0), modTime: time.Unix(0, 0)}
```

Comment:
looking at openshift repos, this is consistent, but I'm not sure why the build-machinery-go bindata.mk results in this: 
* https://github.com/openshift/cluster-authentication-operator/blob/master/pkg/operator/assets/bindata.go#L196 
* https://github.com/openshift/cluster-openshift-apiserver-operator/blob/master/pkg/operator/v311_00_assets/bindata.go#L126

Reply:
I had removed the -nometadata back then to conserve file permissions when writing bindata assets to disk.

The use case was to eventually replace the highly redundant and complex "manifest applier" code for the loaded components with a simple aeevice to render component manifests as kustomization to disk, where the currently existing kustomizer service would find and automatically apply it. You can still see that code in the in the `component-kustomizer-poc` branch.

Reply:
@fzdarsky, what's the use case for customizing the embedded manifests?